src/Sarif.Viewer.VisualStudio/OpenLogFileCommands.cs

```diff
@@ -271,7 +271,7 @@ private async System.Threading.Tasks.Task MenuItemCallbackAsync(object sender, E
             }
         }
 
-        string ConvertSarifProtocol(string inputUrl)
+        private string ConvertSarifProtocol(string inputUrl)
```

Comment:
static?

Reply:
I would argue that both of these methods (this one and the one below) should be in some utilities class. Am I mistaken?

---
In reply to: [519462594](https://github.com/microsoft/sarif-visualstudio-extension/pull/259#discussion_r519462594) [](ancestors = 519462594)

Reply:
Yes, both static.

"Mistaken" is putting it a bit strongly. But both of these are strongly related to opening files, and there isn't another client at the moment. I'd leave them here.

---
In reply to: [519462594](https://github.com/microsoft/sarif-visualstudio-extension/pull/259#discussion_r519462594) [](ancestors = 519462594)